stable/redis/templates/metrics-svc.yaml

```diff
@@ -14,6 +14,10 @@ metadata:
 {{- end }}
 spec:
   type: {{ .Values.metrics.service.type }}
+  {{ if eq .Values.metrics.service.type "ClusterIP" -}} {{ if .Values.metrics.service.ClusterIP -}}
```

Comment:
can't you use `and` and use a single `if` statement?

```
{{- if and (eq .Values.metrics.service.type "ClusterIP") (.Values.metrics.service.ClusterIP) }}
    ClusterIP: {{ .Values.metrics.service.ClusterIP }}
{{- end }}
```

Reply:
i have done it for loadbalanser too